src/Settings/settings.json

```
@@ -292,6 +292,18 @@
         "param": "QStringList {\"Light\", \"Dracula\", \"Monokai\", \"Solarized\", \"Solarized Dark\"}",
         "tip": "The syntax highlight theme of the code editor"
     },
+    {
+        "name": "Terminal Program",
```

```suggestion
        "name": "Detached Run Terminal Program",
        "desc": “Terminal Program"
```

You should point out it's for detached run in the name.

If you want to omit "Detached Run" in the UI, you can set the "desc" attribute. However, I'd rather not to omit it.

---

Isn't it implied the location of the setting.

Actions -> Detached Execution

"Terminal Program" will be detached terminal.

However, I accept the changes, Maybe we can add in future inbuild terminal and that will conflict with this.

---

> Isn't it implied the location of the setting.
> 
> Actions -> Detached Execution

"Actions -> Detached Execution" is only in the preferences window, not with the setting itself. You don't see "Actions -> Detached Execution" when using `getTerminalProgram". And now the settings are independent of the preferences window.